src/modules/account/components/account-deposit/account-deposit.jsx

```diff
@@ -42,12 +43,12 @@ export default class AccountDeposit extends Component {
   }
 
   render() {
-    const { address, openZeroExInstant } = this.props;
+    const { address, openZeroExInstant, augurNodeNetworkId } = this.props;
     const styleQR = {
       height: "auto",
       width: "100%"
     };
-
+    const show0xInstant = [1, 42].includes(augurNodeNetworkId);
```

Comment:
Can you obtain these values from NETWORK_NAMES instead? Recommend not hard-coding them.